src/main/java/com/kuit/healthmate/domain/supplement/SupplementChecker.java

```diff
@@ -0,0 +1,29 @@
+package com.kuit.healthmate.domain.supplement;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDate;
+
+@Entity
```

Comment:
```suggestion
@Entity
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
위와 같이 일반적으로 entity 클래스의 경우 접근 레벨을 protected로 하는 것이 좋습니다. 외부 패키지에서 해당 클래스가 생성되거나 하면 곤란하겠죠. 엔티티는 Jpa에서 정말 정말 중요한 친구이니 항상 조심해야하는거 같아요.